sdk/core/azure-core-tracing-opentelemetry/src/samples/CreateConfigurationSettingTracingSample.md

```diff
@@ -28,19 +28,19 @@ Sample uses **[opentelemetry-sdk][opentelemetry_sdk]** as implementation package
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-core-tracing-opentelemetry</artifactId>
-    <version>1.0.0-beta.4</version>
+    <version>1.0.0-beta.6</version>
```

Comment:
@JimSuplizio are we not processing version bumps sample md files? (Assuming they get correctly annotated in with tags). I cannot remember if we still have the md file processing disabled by default.

Reply:
@weshaggard we do if it's tagged and they're not running the script with the --sr option. These aren't tagged.